light-client-verifier/Cargo.toml

```diff
@@ -23,14 +23,27 @@ all-features = true
 rustdoc-args = ["--cfg", "docsrs"]
 
 [features]
+std = [
+  "tendermint/std",
+  "serde/std",
+  "time/std",
+  "flex-error/std",
+]
 default = ["flex-error/std", "flex-error/eyre_tracer"]
+secp256k1 = ["tendermint/secp256k1"]
+test-helpers = [ "sp-core"]
 
 [dependencies]
 tendermint = { version = "0.24.0-pre.2", path = "../tendermint", default-features = false }
 derive_more = { version = "0.99.5", default-features = false, features = ["display"] }
 serde = { version = "1.0.106", default-features = false }
 time = { version = "0.3.5", default-features = false }
 flex-error = { version = "0.4.4", default-features = false }
+sp-std = { version = "4.0.0", default-features = false }
+sp-core = { version = "6.0.0", features = ["full_crypto"], optional = true }
```

Comment:
Are these dependencies really necessary?

Reply:
yes its used for tests and std end-users

Reply:
Can we instead use `core` and `alloc` crates? We would rather avoid introducing Substrate-specific dependencies into the light client.